styles/style.css

```diff
@@ -445,3 +445,194 @@ p {
   cursor: pointer;
   margin: 18px;
 }
+
+@media (min-width: 992px) {
+  .navbar {
+    padding: 1.5rem 8rem;
+  }
+
+  .navbar__menu {
+    display: flex;
+    align-items: center;
+  }
+
+  .navbar__menu__ul {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+  }
+
+  .navbar__menu__li {
+    margin: 0 16px;
+  }
+
+  .navbar__hamb {
+    display: none;
+  }
+
+  .head::before {
+    background-image: url(../images/desktop/Header-bg@2x.png);
+    padding: 6rem;
+    border-bottom-left-radius: 8rem;
+  }
+
+  .head {
+    padding: 10rem;
+    padding-right: 15rem;
+  }
+
+  .head__title span {
+    width: 15ch;
+  }
+
+  .head__link {
+    margin: 0.7rem 0;
+  }
+
+  .works__cards {
+    padding: 5rem;
+  }
+
+  .works__card {
+    max-width: 1104px;
+    -moz-column-gap: 46px;
+    column-gap: 46px;
+    padding: 1.5rem;
+  }
+
+  .works__card__text {
+    padding: 1rem;
+  }
+
+  .works__card > * {
+    flex-basis: 50%;
+  }
+
+  .works__card:nth-child(1),
+  .works__card:nth-child(3) {
+    flex-direction: row;
+  }
+
+  .works__card:nth-child(2),
+  .works__card:nth-child(4) {
+    flex-direction: row-reverse;
+  }
+
+  .about {
+    padding: 6rem;
+    flex-direction: row;
+    -moz-column-gap: 37px;
+    column-gap: 37px;
+    border-top-right-radius: 8rem;
+  }
+
+  .about__myself {
+    flex-basis: 25rem;
+  }
```

Comment:
- [ ] As requested by the previous reviewer, Kindly fix the margin of the technology categories (languages, frameworks, and skills) and their arrows so as to follow the Figma design.
your design
![Capture](https://user-images.githubusercontent.com/85095702/160025714-f68a56e9-1d4c-47a1-87da-bca620f7653e.PNG)

Reply:
I have implemented the suggestions. @Kingstalux  Please can you give me some feedback before I resubmit? As this is my last chance. Thanks a lot.

Reply:
> * [x]  As requested by the previous reviewer, Kindly fix the margin of the technology categories (languages, frameworks, and skills) and their arrows so as to follow the Figma design.
>       your design
>       ![Capture](https://user-images.githubusercontent.com/85095702/160025714-f68a56e9-1d4c-47a1-87da-bca620f7653e.PNG)

Suggested change implemented. Thanks a lot.

Reply:
Hello @ONJoseph, 
The arrows are okay now but you've removed the white background from the about section. Kindly add back the white background and you'll be good to go.
![Annotation 2022-03-25 052514](https://user-images.githubusercontent.com/85113567/160054085-fcccbc2e-3cd1-4a95-9eae-192768e7dffa.jpg)

Reply:
I have implemented the suggestion. @Kingstalux Please is it good to go now? As this is my last chance. Thanks a lot.

Reply:
Hello @ONJoseph ,
- The white background is supposed to extend right up to the contact section and the top right of the about section should be curved.
- Also, there is to be some space between the top of the about section and the about heading. 
- Your design:
![Annotation 2022-03-25 201500](https://user-images.githubusercontent.com/85113567/160186973-ec4c462b-78ae-49f5-a540-189a963d9073.jpg)
- Figma design:
![Annotation 2022-03-25 201441](https://user-images.githubusercontent.com/85113567/160186994-11e18d06-7e62-4af6-beb6-70f4f9de73b1.jpg)

Reply:
Grateful for the kind feedback @Kingstalux . I have tried to effect the change requested. Please can I proceed to submit for final approval? As this is my last submission chance. Thanks.

Reply:
yeah, you can submit now for approval 👍